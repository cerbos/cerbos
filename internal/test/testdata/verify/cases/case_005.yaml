description: Invalid test suite with invalid suite definition

