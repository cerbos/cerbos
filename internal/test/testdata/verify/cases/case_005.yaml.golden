{
  "suites": [
    {
      "file": "invalid_test.yaml",
      "name": "Unknown",
      "summary": {
        "overallResult": "RESULT_ERRORED"
      },
      "error": "failed to load test suite: invalid TestSuite.Tests: value must contain at least 1 item(s)"
    }
  ],
  "summary": {
    "overallResult": "RESULT_ERRORED"
  }
}